### PR TITLE
Add skeleton test for test_multiple_volumes_creation_with_degraded_availability

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#2073

### CODE REVIEW

File: manager/integration/tests/test_basic.py [hunk 1]
```diff
@@ -3234,6 +3234,30 @@ def test_allow_volume_creation_with_degraded_availability_error(
...
+    same time.
+
+    Steps:
+    1. create StorageClass longhorn-extra with numberOfReplicas=5
+       Set allow-volume-creation-with-degraded-availability to True
+    2. Deploy this StatefulSet:
+       https://github.com/longhorn/longhorn/issues/2073#issuecomment-742948726
+    3. In a 1-min retry loop, Verify that all 10 volumes are healthy
+    4. Delete the StatefulSet
+    5. In a 1-min retry loop, Verify that all 10 volumes are deachted
```
Comment (yasker) [inline manager/integration/tests/test_basic.py:3252]: `detached`

File: manager/integration/tests/test_basic.py [hunk 2]
```diff
@@ -3234,6 +3234,30 @@ def test_allow_volume_creation_with_degraded_availability_error(
...
+    6. Find and delete the PVC of the 10 volumes.
+    7. In a 1-min retry loop, Verify that all 10 volumes are deleted
+    8. Reset Set allow-volume-creation-with-degraded-availability to False
```
Comment (yasker) [inline manager/integration/tests/test_basic.py:3255]: Reset will be done by the finalizer automatically.

We should make finalized clean up the extra storage classes too.

**Review by yasker — CHANGES_REQUESTED** [review]

File: manager/integration/tests/test_basic.py (hunk 2 above)
Comment (PhanLe1010) [reply manager/integration/tests/test_basic.py]: Thank you. Fixed

File: manager/integration/tests/test_basic.py (hunk 1 above)
Comment (PhanLe1010) [reply manager/integration/tests/test_basic.py]: Thank you. Fixed

**Review by yasker — APPROVED** [review]

**Pull request merged**